backend/src/test/java/com/github/redstylzz/backend/model/TestDataProvider.java

```diff
@@ -78,10 +80,12 @@ public static Payment testPayment(String paymentID,
                 .build();
     }
 
-    public static UUID mockUUID() {
-        UUID randUUID = UUID.randomUUID();
+    public static String UUID_STRING = "06f1eb01-cdaf-46e4-a3c8-eff2e4b300dd";
```

Comment:
you may want to consider making this final as public static values of a class should not be changed